pkg/recommender.py

```diff
@@ -686,39 +655,62 @@ def _schedule_evt(self):
                             # reset scheduled events
                             schedule_evts[1] = (evening_timedelta, 'evening message')  # since tuples immutable
 
+
+                #send the blank message after everything for both morning and evening messages-------------
+                _ = call_ema('1', '995', alarm='false', test=self.test_mode) #send directly even if stop questions
+
+
                 log(f'Scheduled event sent: {event_id}', timer=self.timer)
 
             except Exception as error:
                 log('Send scheduled action error:', error, timer=self.timer)
```

Comment:
Hey! Meiyi created a program to send emails when an error appears. I just want to confirm, should I call her program everywhere we have an except statement since these are the only places where we can detect an error? Is this too repetitive and inefficient? Thanks for the help!